src/sdk/auth/browser-auth-provider.ts

```diff
@@ -19,7 +19,7 @@ export class IlandBrowserAuthProvider implements AuthProvider {
       realm: DEFAULT_REALM
     };
     this._keycloak = Keycloak(kcConfig);
-    this._tokenObservable = Observable.create((observable: Subscriber<string>) => {
+    this._tokenObservable = new Observable<string>((observable: Subscriber<string>) => {
```

Comment:
why?

Reply:
Because Circle CI was throwing an error about this deprecated method
<img width="1640" alt="Capture d’écran 2019-10-30 à 14 43 19" src="https://user-images.githubusercontent.com/1576511/67863525-5a117a00-fb24-11e9-93bc-b399f5409ec3.png">